doc/project/code_flow.rst

```diff
@@ -6,38 +6,58 @@ Code Flow and Branches
 Introduction
 ************
 
-The zephyr Git repository has three types of branches:
+The Zephyr Git repository has three types of branches:
 
 main
   Which contains the latest state of development
 
-topic-\*
-  Topic branches that are used for shared development of a new feature
+collab-\*
+  Collaboration branches that are used for shared development
+  of new features to be introduced into main branch when ready. Creating a new
+  collaboration branch requires a justification and a TSC approval. Collaboration branches
+  shall be based off the main branch and any changes developed in the collab
+  branch shall target the main development branch. For released versions of
+  Zephyr, the introduction of fixes and new features, if approved by the TSC,
+  shall be done using backport pull requests.
 
 vx.y-branch
   Branches which track maintenance releases based on a major
   release
 
-Development in topic branches before features go to mainline allows teams to
+Development in collaboration branches before features go to mainline allows teams to
```

Comment:
The PR is inconsistently using "Collaboration" (capitalized) and not "collaboration"; please pick one and be consistent